### PR TITLE
[IA-2322] Remove miniconda to fix terminal

### PR DESCRIPTION
**Pull request by jdcanas**
The versions match in terminal and in a notebook when tested locally.

### CODE REVIEW

File: updateVersions.sc [hunk 1]
```diff
@@ -1,6 +1,6 @@
-val workbenchUtil2 = "0.10-c3ef6b80-SNAP"
+// Must be run with scala 2.13 or above.
 
-interp.configureCompiler(_.settings.YpartialUnification.value = true)
```
Comment (jdcanas) [inline updateVersions.sc:3]: This line causes the script to fail if you run with scala 2.13. The feature is now enabled by default. https://stackoverflow.com/questions/61035986/value-ypartialunification-is-not-a-member-of-scala-tools-nsc-settings

File: updateVersions.sc [hunk 2]
```diff
@@ -1,6 +1,6 @@
-val workbenchUtil2 = "0.10-c3ef6b80-SNAP"
+// Must be run with scala 2.13 or above.
```
Comment (Qi77Qi) [inline updateVersions.sc:1]: I think comment should be `must be run `ammonite for Scala 2.13+`

**Review by Qi77Qi — APPROVED** [review]

**Comment by rtitle** [comment]
I think the aou image build failed as a result of this: https://fc-jenkins.dsp-techops.broadinstitute.org/job/leonardo-build-terra-docker/1528/console

May want to confirm with AoU their use of conda

**Comment by rtitle** [comment]
Also may want to revert in the interim and delete the images that were built in GCR

**Pull request merged**